src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40;
+  const totalRentPrice = days * rentCost;
+  const longTermRent = 7;
+  const longTermDiscount = 50;
+  const smallTermRent = 3;
+  const smallTermDiscount = 20;
+
+  if (days < smallTermRent) {
+    return totalRentPrice;
+  }
+
+  if (days >= smallTermRent && days <= 6) {
+    return totalRentPrice - smallTermDiscount;
+  }
```

Comment:
The condition `days <= 6` is redundant because if `days` is 7 or more, the third condition will be executed due to the return statement in the second condition. You can simplify the code by removing this check.